### PR TITLE
Add _build_options() to FixedAddressV4

### PR DESCRIPTION
**Pull request by arnaudcoquelet**
Add the ability to fetch the DHCP options associated to a FixedAddressV4

### CODE REVIEW

File: infoblox_client/objects.py [hunk 1]
```diff
@@ -742,13 +742,21 @@ def ip(self, ip):
 
 class FixedAddressV4(FixedAddress):
     _infoblox_type = 'fixedaddress'
-    _fields = ['ipv4addr', 'mac', 'network_view', 'extattrs', 'network']
+    _fields = ['ipv4addr', 'mac', 'network_view', 'extattrs', 'network','options']
```
Comment (bondar-pavel) [inline infoblox_client/objects.py]: nit: need a space after comma to make pep8 checks happy, also this line is longer than 79 characters, so need to move 'options' to next line.

> infoblox_client/objects.py:745:72: E231 missing whitespace after ','
> infoblox_client/objects.py:745:80: E501 line too long (82 > 79 characters)

Just run 'make lint' to make sure your changes passes pep8/flake8 rules.

P.S. shame on me that I still have not included flake checks into Travis CI (created #79 for that).

File: infoblox_client/objects.py [hunk 2]
```diff
@@ -742,13 +742,21 @@ def ip(self, ip):
...
     _remap = {'ipv4addr': 'ip'}
     _ip_version = 4
 
+    @staticmethod
+    def _build_options(members):
+        if not members:
+            return None
+        return [DhcpOption.from_dict(m) for m in members]
+
+    _custom_field_processing = {'options': _build_options.__func__}
```
Comment (bondar-pavel) [inline infoblox_client/objects.py]: +1, that is correct usage of custom field processing rules.

**Comment by bondar-pavel** [comment]
Overall PS looks good, have just one nit about making pep8 checks happy.
Also no UTs are included for that, but I realized there is no good example in test_objects, so I will add one by myself once it get merged.

**Comment by arnaudcoquelet** [comment]
I made your recommended changes.

**Comment by bondar-pavel** [comment]
@arnaudcoquelet Thanks! We are almost there.
I see two commits (second just remove space in the end of the line).
Could you please squash these two commits into one?

**Comment by arnaudcoquelet** [comment]
I'm sorry, I'm new to git and the merge....

**Comment by arnaudcoquelet** [comment]
I tried to squash the 2 commits in one, let me know if that works now.
Sorry for those backs and forths....

**Comment by bondar-pavel** [comment]
@arnaudcoquelet Now it looks good, thanks for your efforts!
I remember my first attempt to squash commits (well it took a while :)), so that is totally ok.
Now I'll do some manual verifications with real Grid and plan to merge it in comming hour.
New release of infoblox-client is planned for today, so these changes most likely to be available with infoblox-client 0.4.2

**Comment by bondar-pavel** [comment]
@arnaudcoquelet Sorry, I have missed the fact that you still have two commits.
Squashing commits never looks simple the first time, espesially because there are a lot of ways to do it.

In your case I would recommend to do:

    git reset --soft HEAD~2

It deletes 2 last commits, but leaves all the changes in uncommitted stage, and. you can verify them with:

    git diff

After that you need to create new commit instead of 2 old commits.

    git commit -a 

Then verify you have only one your commit:

    git log

If everything is correct, updete this pull request by push with force:

    git push -f

**Comment by arnaudcoquelet** [comment]
Ok, now git log shows only one commit ;-)

I also added the fields network and status to the IPAddress class.
That allow to lookup for IPAddresses for a specific network.

**Comment by bondar-pavel** [comment]
Hi @arnaudcoquelet! Sorry for delay in response, I had to wrap up several priority tasks before leaving to vacaction.
Code looks good and now it is a single commit, thanks for your effort:)
Made some quick tests of PR with Grid, looks workable, so merging.

**Comment by arnaudcoquelet** [comment]
Great thanks.

Arnaud Coquelet
Nokia

**Comment by bondar-pavel** [comment]
@arnaudcoquelet FYI, this changes are now released and available in infoblox-client 0.4.3

**Comment by arnaudcoquelet** [comment]
Great thanks

**Pull request merged**